### PR TITLE
Tweaked Banner

### PR DESCRIPTION
**Pull request by mihir-putcha**
Screenshot 2024-09-01 at 10 01 25 PM

Reduced banner height, made entire box clickable, removed first sentence, underlined here, added red = "noopener noreferrer" attribute

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,9 +82,11 @@
 </head>
 
 <body>
-  <div id="banner">
-    <p>Interested in joining the team behind Cal Hacks? Apply to be a director <a href="https://bit.ly/hack-fa24-app" target="_blank">here</a> by 9/11 @ 11:59 PM PST!</p>
-  </div>
+  <a href="https://bit.ly/hack-fa24-app" target="_blank" id="banner-link">
+    <div id="banner">
+      <p>Apply to be a director here by 9/11 @ 11:59 PM PST!</p>
```
Comment (aryanvichare) [inline index.html]: Underline the "here" to give the illusion of a link.

File: index.html [hunk 2]
```diff
@@ -82,9 +82,11 @@
 </head>
 
 <body>
-  <div id="banner">
-    <p>Interested in joining the team behind Cal Hacks? Apply to be a director <a href="https://bit.ly/hack-fa24-app" target="_blank">here</a> by 9/11 @ 11:59 PM PST!</p>
-  </div>
+  <a href="https://bit.ly/hack-fa24-app" target="_blank" id="banner-link">
```
Comment (aryanvichare) [inline index.html]: Add property `rel="noopener noreferrer"` to the `<a>` tag

File: index.html (hunk 1 above)
Comment (aryanvichare) [reply index.html]: Just underline the "here" and nothing else

File: package-lock.json [hunk 3]
```diff
@@ -39,11 +39,146 @@
         "node": ">= 8"
       }
```
Comment (aryanvichare) [inline package-lock.json]: Do we need `package-lock.json` changes?

Comment (mihir-putcha) [reply package-lock.json]: i don't think so - reverted the changes

**Pull request merged**